openshift/templates/nsp/build.yaml

```diff
@@ -0,0 +1,118 @@
+---
+kind: Template
+apiVersion: v1
+metadata:
+  name: global-nsp-build-template
+objects:
+  - kind: NetworkPolicy
+    apiVersion: networking.k8s.io/v1
+    metadata:
+      name: deny-by-default
+    spec:
+      description: |
+        Deny all traffic by default.
+      podSelector: {}
+      ingress: []
+
+  - kind: NetworkSecurityPolicy
+    apiVersion: security.devops.gov.bc.ca/v1alpha1
+    metadata:
+      name: any-to-any
+    spec:
+      description: |
+        Disable Aporeto policies - Allow all pods within the namespace to communicate.
+      source:
+        - - $namespace=${NAMESPACE_NAME}-${ENV_NAME}
+      destination:
+        - - $namespace=${NAMESPACE_NAME}-${ENV_NAME}
+
+  - kind: NetworkSecurityPolicy
+    apiVersion: security.devops.gov.bc.ca/v1alpha1
+    metadata:
+      name: any-to-external
+    spec:
+      description: |
+        Disable Aporeto policies - Allow all pods within the namespace full access to external systems.
+      source:
+        - - $namespace=${NAMESPACE_NAME}-${ENV_NAME}
+      destination:
+        - - ext:network=any
```

Comment:
These should go, they are unnecessary/unsupported now

Reply:
removed